packages/foreman/foreman/foreman.spec

```diff
@@ -1144,6 +1145,17 @@ if [ ! -f %{_datadir}/%{name}/config/initializers/local_secret_token.rb ]; then
   chgrp foreman %{_datadir}/%{name}/config/initializers/local_secret_token.rb
   %{foreman_rake} security:generate_token >/dev/null 2>&1 || :
   chmod 0640 %{_datadir}/%{name}/config/initializers/local_secret_token.rb
+  mv %{_datadir}/%{name}/config/initializers/local_secret_token.rb %{_sysconfdir}/%{name}/
```

Comment:
If we ensure the symlink is in place before running the token generation, would that also ensure it's generated in the right place? Ideally that means we can ship a symlink inside the package. We'd need to ensure the current token isn't overwritten though.